apps/front/src/app/header/header.component.html

```diff
@@ -1,43 +1,166 @@
-<nav class="navbar navbar-expand-sm sticky-top bg-white fs-5 shadow-sm" [hidden]="hidden">
+<scholarsome-resend-email
+  *ngIf="!this.verificationResult && this.signedIn"
```

Comment:
`verificationResult` needs to be explicitly checked whether it is `false`, not **falsey** as it will always be false if there is no cookie.